### ISSUE TITLE
ACI deployment issue

### ISSUE THREAD

**Issue by elenamoor** [MicrosoftLearning/mslearn-dp100#89]
Dear team!

I am not 100% sure if the issue I'm describing belongs here. If not please point me to where it should be reported to. 

This week and previous weeks we have seen a pattern where at least a third of the students were are able to properly deploy models to Azure Container Instances using Azure Passes in multiple regions. For some students it works with no issues. 

Behavior: The container was "deployed" (meaning, it showed up in the ACI), then after a extended time it would show as running, but it never initialized, then showed as unhealthy, then showed as not running. In the mean time, trainers and other MTTs were not able to replicate the issue (everything worked fine for them in internal AIRS, Azure Pass and Visual Studio subscription)

This issue also reported to occur in DP-090. Some trainers report it to occur ONLY with Azure passes and a similar problem with CloudSlice in DP-090. 

I am also creating a ticket with Cloudslice from the vm environment just in case.

Thank you
Elena Moor

**Comment by GraemeMalcolm** [comment]
The most likely cause of this is dynamic resource constraints applied to low-priority subscriptions (esp. Azure Pass). Unfortunately, there's not much we can do about that.
For Cloud Slice, we've seen issues when provisioning Azure Databricks clusters (which doesn't always seem to honor the available quota metrics returned by the API). Issues related to ACI deployments on cloud slice have generally been caused by students not selecting *exactly* the configuration described in the lab step (which is enforced by a policy). If you're seeing otherwise, can you please record as much detail as possible about the code or steps used to provision the ACI and the error message received?

**Comment by elenamoor** [comment]
Yeah I agree with that. The problem is that I can not replicate the issue myself in any shape under any subscription. Thank you for your response, I will ask to record as many details / screenshots as possible.

**Comment by GraemeMalcolm** [comment]
![image](https://user-images.githubusercontent.com/11771134/183106272-af050b23-b190-4cee-8e55-328043f2ef79.png)

**Comment by zeeb33** [comment]
I have not been able to retrieve any logs running that command. 
![image](https://user-images.githubusercontent.com/42649591/183671566-b96b6f22-4348-4443-b59d-240926f926fc.png)

**Comment by gcoyle83** [comment]
I am running the 'Create a real-time inferencing service' notebook in an AzureML workspace under an enterprise account and encounter the following error: 
```
Error:
{
  "code": "AciDeploymentFailed",
  "statusCode": 400,
  "message": "Aci Deployment failed with exception: Error in entry script, ModuleNotFoundError: No module named 'applicationinsights', please run print(service.get_logs()) to get details.",
  "details": [
    {
      "code": "CrashLoopBackOff",
      "message": "Error in entry script, ModuleNotFoundError: No module named 'applicationinsights', please run print(service.get_logs()) to get details."
    }
  ]
}
```
When I run `print(service.get_logs())` I get the same `None` return value indicated by the above post.

**Comment by madiepev** [comment]
Using a curated environment instead of a custom environment to use a newer Python version seems to fix the issue for now. Please let me know if the problem persists.
